pydra/engine/tests/test_task.py

```diff
@@ -986,6 +987,28 @@ def testfunc(a: int, b: float = 0.1) -> ty.NamedTuple("Output", [("out", float)]
     assert (tmpdir / funky.checksum / "messages.jsonld").exists()
 
 
+def test_audit_task(tmpdir):
+    @mark.task
+    def testfunc(a: int, b: float = 0.1) -> ty.NamedTuple("Output", [("out", float)]):
+        return a + b
+
+    from glob import glob
+
+    funky = testfunc(a=2, audit_flags=AuditFlag.PROV, messengers=FileMessenger())
+    funky.cache_dir = tmpdir
+    funky()
+    message_path = tmpdir / funky.checksum / "messages"
+    # go through each jsonld file in message_path and check if the label field exists
+    for file in glob(str(message_path) + "/*.jsonld"):
+        with open(file, "r") as f:
+            data = json.load(f)
+            if "label" in data:
+                print(data)
+                assert True
```

Comment:
I don't think this test can ever fail. The test should be written in a way that can have `assert False`, e.g. `assert "label" in data`